lib/viewmodels/user_posts_view_model.dart

```diff
@@ -19,7 +20,14 @@ class UserPostsViewModel extends AutoDisposeAsyncNotifier<UserPostsState> {
     final user = ref.watch(validLoggedInUserIdProvider);
 
     final postsFirestore = await postRepository.getUserPosts(user);
-    final posts = postsFirestore.map((post) {
+
+    // Sort the posts by publication time from lattest to oldest
```

Comment:
```suggestion
    // Sort the posts by publication time from latest to oldest
```
small typo

Reply:
Thanks for the catch :+1: 

Fix in 3b0d4cd8545cfdd067b5599deb827577b66de9b5.